server/pulp/server/db/connection.py

```diff
@@ -8,6 +8,12 @@
 from gettext import gettext as _
 
 import mongoengine
+
+try:
+    from mongoengine.connection import MongoEngineConnectionError
+except ImportError:
+    from mongoengine.connection import ConnectionError as MongoEngineConnectionError
```

Comment:
Maybe a comment that this import is for mongoengine < 0.11 will be useful, what do you think?